Makefile

```diff
@@ -69,16 +73,24 @@ endif
 
 ifeq ($(TARGET_NAME),TARGET_NANOS)
 DEFINES   += IO_SEPROXYHAL_BUFFER_SIZE_B=128
-DEFINES   += HAVE_UX_FLOW
 else
 DEFINES   += IO_SEPROXYHAL_BUFFER_SIZE_B=300
-DEFINES   += HAVE_GLO096 HAVE_UX_FLOW
-DEFINES   += HAVE_BAGL BAGL_WIDTH=128 BAGL_HEIGHT=64
+endif
+
+ifeq ($(TARGET_NAME),TARGET_STAX)
+DEFINES   += HAVE_NBGL
```

Comment:
HAVE_NBGL is not required anymore here in latest SDK versions. Defined in `Makefile.defines` of the SDK.

Reply:
Done :+1: